Source/RunActivity/Viewer3D/Shapes.cs

```diff
@@ -1002,7 +1002,7 @@ public override void PrepareFrame(RenderFrame frame, ElapsedTime elapsedTime)
     public class RoadCarShape : AnimatedShape
     {
         public RoadCarShape(Viewer viewer, string path)
-            : base(viewer, path, new WorldPosition(), ShapeFlags.ShadowCaster)
+            : base(viewer, path, new WorldPosition())
```

Comment:
This is wrong; the default shape flags is `None`.

Reply:
why woulnd't a roadcar be allowed to cast shadows?

Reply:
Road cars should have shadows. This change is **removing** shadows from road cars, by removing the `ShadowCaster` flag. 🙂

Reply:
ah right, just ignore above comment, the code was already updated